website/www/site/content/en/documentation/sdks/java-multi-language-pipelines.md

```diff
@@ -226,19 +232,70 @@ For example, to start the standard expansion service for a Python transform,
 [ExpansionServiceServicer](https://github.com/apache/beam/blob/master/sdks/python/apache_beam/runners/portability/expansion_service.py),
 follow these steps:
 
-1. Activate a Python virtual environment and install Apache Beam, as described
-   in the [Python quick start](/get-started/quickstart-py/).
-2. In the **beam/sdks/python** directory of the Beam source code, run the
-   following command:
+1. Activate a new virtual environment. See [here](https://beam.apache.org/get-started/quickstart-py/#create-and-activate-a-virtual-environment)
```

Comment:
Maybe use this: "1. Activate a new virtual environment following [these instructions](https://beam.apache.org/get-started/quickstart-py/#create-and-activate-a-virtual-environment)."